datalab/utils/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 """Google Cloud Platform library - Internal Helpers."""
 
-from ._async import async, async_function, async_method
+from ._async import async_, async_function, async_method
```

Comment:
Thanks for incorporating the naming style from https://github.com/googledatalab/pydatalab/pull/728. It is shorter and a more standard pattern.

Reply:
Thanks you very much for your support.